src/components/play-pause/play-pause.js

```diff
@@ -62,11 +62,19 @@ class PlayPause extends Component {
           <button
             tabIndex="0"
             aria-label={
-              <Text id={props.isPlaybackEnded ? 'controls.startOver' : this.props.isPlayingAdOrPlayback ? 'controls.pause' : 'controls.play'} />
+              <Text
+                id={
+                  props.isPlaybackEnded && !this.props.adBreak
```

Comment:
extract to const isStartOver to be used also in line 77

Reply:
done